### ISSUE TITLE
Example - Triangles - DataType indice error

### ISSUE THREAD

**Issue by zentol** [stratosphere/stratosphere#722]
The defined indices in the EdgeWithDegrees class don't match the indizes used in the constructor. If you were to group over V2, you would use the first degree instead of the second vertex if the edge was build with the constructor.
https://github.com/stratosphere/stratosphere/blob/master/stratosphere-examples/stratosphere-java-examples/src/main/java/eu/stratosphere/example/java/triangles/util/EdgeDataTypes.java

```
public static class EdgeWithDegrees extends Tuple4<Integer, Integer, Integer, Integer> {
    public static final int V1 = 0;
    public static final int V2 = 1;
    public static final int D1 = 2;
    public static final int D2 = 3;

    public EdgeWithDegrees(final Integer vertex1, final Integer degree1, final Integer vertex2, final Integer degree2) {
        super(vertex1, degree1, vertex2, degree2);
    }
...
```

**Comment by fhueske** [comment]
Oh, yes, you're right!
The constructor isn't used in the code, that's probably why the bug did not cause the tests to fail.
We could simply remove it...

**Comment by StephanEwen** [comment]
Fix provided by #722
